### ISSUE TITLE
Update to 4.8.1 causes account to be wiped

### ISSUE THREAD

**Issue by pjotrek-b** [floccusaddon/floccus#971]
After updating from 4.7.0 to 4.8.1 my account information was completely gone.

I've performed the same update from 4.7.0 to 4.8.0 without any issues a few days ago on another machine.
I wasn't surprised to see the most recent issue (#970) seems to report also losing account information.

I've created a separate issue, because #970 seems to describe a different set of steps causing it.

1. Update Firefox addon from 4.7.0 to 4.8.1
2. Click on the Floccus icon (as usual)
3. You see no existing accounts, just "+ new Account"

Thank you in advance!

**Comment by marcelklehr** [comment]
Did you set a password for floccus?

**Comment by marcelklehr** [comment]
@pjotrek-b Can you think of any differences between the setups on the two machines?

**Comment by imforumman** [comment]
I second this. Same behavior here. If I re-add the account, it correctly authenticates, adds the account, but doesn't sync. Next time I open the little window, the account is gone again.

It should be noted, that the usual Text "There is no account here..." is missing. I see that text above the colored "add account" button only after a clean install.

**Comment by pjotrek-b** [comment]
Sorry for not completing the template.

  * Yes, I have a password set.
  * The one where the account disappears is Xubuntu 18.04, the other one (no problems) 20.04.
  * But I haven't tried/dared to update the working machine from 4.8.0 to 4.8.1...

**Comment by marcelklehr** [comment]
Please try v4.8.2
